scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -59,12 +59,23 @@ FUNDS_PER_BYRON_ADDRESS=$((${FUNDS_PER_GENESIS_ADDRESS} - 1000000))
 NETWORK_MAGIC=42
 SECURITY_PARAM=10
 
-OS=$(uname -s) DATE=
-case $OS in
-  Darwin )       DATE="gdate";;
-  * )            DATE="date";;
+UNAME=$(uname -s) DATE=
+case $UNAME in
+  Darwin )      DATE="gdate";;
+  Linux )       DATE="date";;
+  MINGW64_NT* ) UNAME="Windows_NT"
+                DATE="date";;
 esac
 
+sprocket() {
```

Comment:
I'd like to see some documentation explaining why we need to do this. Based on your changes and the name of the PR, it looks like we cant just use a file path for the socket? I have no idea what that path `\\.\pipe\` refers to on windows. Is this some special path for pipes (or socket equivalent for windows)? I assume forward slashes really don't work?

Reply:
Windows is weird.

So on Windows, we're normally used to paths looking like this: `C:\MyDirectory\MyFile`, but this is only what the user sees.

Filenames actually look like this to the operating system filesystem: `\\?\C:\MyDirectory\MyFile` and `\\.\C:\MyDirectory\MyFile`

This way of writing filenames is called the UNC Path or the device path and allows you to refer to things that are not files.  For example named pipes.

So long story short, Windows does not follow the Unix philosophy where everything is a file.  Instead the devices and non-file resources are referenced via a schema that embeds the filesystem but reserves other namespaces for non-files.

Reply:
I've added a short explanation as a comment to the script.

Reply:
Amazing thanks!